### DISCUSSION TITLE
after creating jenkins job role i used to give credentials but it shows like

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1960] [Q&A]
![Screenshot (57)](https://github.com/user-attachments/assets/dc7fc58e-49b2-419c-8636-23657c11f939)

**Comment by chaithanya1812** [comment]
Hello @Bhashitha19, you can give any name you want. 
It showing id (ssh) already present, so give another name like ssh1 or workstation_ssh ... And use that id

**Reply by Bhashitha19** [reply]
k sir

**Reply by Bhashitha19** [reply]
![Screenshot (58)](https://github.com/user-attachments/assets/78480f3c-955d-4085-91d1-4a095b8186ca)
